sensor_msgs/msg/Attitude.msg

```diff
@@ -0,0 +1,14 @@
+# This message represents 3D attitude
+
+int8 ENU=1
+int8 NED=2
+int8 ENU_FLOATING=3
+int8 NED_FLOATING=4
```

Comment:
There'd be a comment here explaining what these mean and what their relationship is to `sensor_frame`?